people.json

```diff
@@ -480,4 +480,15 @@
 			}
 		]
 	}
+	{
+		"github": "komodoooo",
+		"name": "Sesso Pazzo",
+		"title": "sus",
```

Comment:
haha just an FYI only titles with a UBC affiliation will go in the in-person exhibition, but I will take this as is for the website!